docs/user/alerting/defining-alerts.asciidoc

```diff
@@ -59,11 +59,19 @@ Each action type exposes different properties. For example an email action allow
 [role="screenshot"]
 image::images/alert-flyout-action-details.png[UI for defining an email action]
 
-Using the https://mustache.github.io/[Mustache] template syntax `{{variable name}}`, you can pass alert values at the time a condition is detected to an action.  Note that using two curly braces will escape any HTML.  Should you need to preserve HTML, use three curly braces (`{{{`). Available variables differ by alert type, and a list can be accessed using the "add variable" button.
+Using the https://mustache.github.io/[Mustache] template syntax `{{variable name}}`, you can pass alert values at the time a condition is detected to an action. Available variables differ by alert type, and the list of available variables can be accessed using the "add variable" button.
 
 [role="screenshot"]
 image::images/alert-flyout-action-variables.png[Passing alert values to an action]
 
+There are some cases where the variable values will be "escaped", when used in a context where escaping is needed:
+
+- for the <<email-action-type, Email>> connector, the `message` action configuration property escapes any characters that would be interpreted as Markdown
+- for the <<slack-action-type, Slack>> connector, the `message` action configuration property escapes any characters that would be interpreted as Slack markdown
+- for the <<webhook-action-type, Webhook>> connector, the `body` action configuration property escapes any characters that are invalid in JSON string values
```

Comment:
I'm not sure I'd understand what _action configuration property_ means if I came in without knowledge of the framework internals.

Perhaps we should just call it the `message` field? Or something like that, as it's a form field which the user inputs at creation, right?

Reply:
heh, ya, I was working from the docs, where action "params" wasn't referred to as params, but a property, and so the resulting word salad was a bit unwieldy.  Will wordsmith that one a bit more, see if I can find other references to a param like that somewhere ...

Reply:
Poking around the docs, action params seem to be referred to as "Action configuration" in the docs, and such references refer to the enclosed "properties":

![image](https://user-images.githubusercontent.com/25117/105195174-5b9fd700-5b08-11eb-9a34-455d7a1b2758.png)

So, unless we want to use some brand new terminology, I think "action configuration property" is probably the right choice.  cc: @gchaps  ?

Reply:
++ to "action configuration property"